gum/backend-linux/gumprocess-linux.c

```diff
@@ -861,9 +865,109 @@ gum_emit_executable_module (const GumModuleDetails * details,
 {
   GumEmitExecutableModuleContext * ctx = user_data;
 
+  if (!gum_maybe_emit_interpreter (details, user_data))
+    return FALSE;
```

Comment:
This return value seems very confusing, I think it would be clearer if we invert it here, and change the second argument to `GumEmitExecutableModuleContext *`.